### ISSUE TITLE
Extension broken with recent updates in Dev channel

### ISSUE THREAD

**Issue by maxim-saplin** [marcglasberg/i18n_extension#66]
```
Target dart2js failed: Exception: ../../../hostedtoolcache/flutter/1.22.0-12.0.pre-dev/x64/.pub-cache/hosted/pub.dartlang.org/i18n_extension-1.5.1/lib/i18n_widget.dart:197:59:
15
Error: No named parameter with the name 'nullOk'.
16
    var newSystemLocale = Localizations.localeOf(context, nullOk: true);
17
                                                          ^^^^^^
18
../../../hostedtoolcache/flutter/1.22.0-12.0.pre-dev/x64/packages/flutter/lib/src/widgets/localizations.dart:413:17:
19
Info: Found this candidate, but the arguments don't match.
20
  static Locale localeOf(BuildContext context) {
21
                ^^^^^^^^

```
The breaking changes might be coming to other channels: 

https://github.com/flutter/flutter/issues/74519

**Comment by danilo108** [comment]
I have the same problem. Is there a quick fix for this?

**Comment by maxim-saplin** [comment]
You can downgrade flutter https://stackoverflow.com/a/65857155/440696

**Comment by rydmike** [comment]
No quick fix in sight with the current incompatibilities between channel stable and beta/dev/master, caused by NNBD.

Heard an indication that something might be in the works, but it was very vague. 

Might be that this is not resolved until NNBD hits stable and everybody must change to "maybeOf" to be able to use latest stable.

Perhaps if "maybeOf" is introduced into stable as a hotfix and "nullOK" is deprecated and everybody can migrate to "maybeOf", then there is at least something users of "nullOK" can do to fix it. Currently there is nothing users of "nullOK" can do to support all channels with the same code base. They can offer forked version of the package, one that uses "nullOK" on stable and "maybeOf" on other news versions/channels of the Flutter SDK, but that is a extra work to do an maintain.

https://github.com/flutter/flutter/issues/74519#issuecomment-770784825

**Comment by maxim-saplin** [comment]
Is it worth creating a null-safe version of the extenison the way it is done with other extensions on pub.dev?

<img width="552" alt="image" src="https://user-images.githubusercontent.com/7947027/106600154-139e9c80-656b-11eb-8954-23832651af84.png">

**Comment by marcglasberg** [comment]
Yes, it is worth doing it, but as far as I know, NNBD will be going to stable soon. I will start updating all my 13 packages as soon as this happens. Until then, I am accepting PRs. If you have the time and knowledge to help update the package, please do.

**Comment by creativecreatorormaybenot** [comment]
You can use:

```
i18n_extension:
  git:
    url: https://github.com/simpleclub-extended/i18n_extension
    ref: maybe-of
```

Until #67 is merged.

**Comment by nerder** [comment]
Hey @marcglasberg any news on when this will be merged?

**Comment by danilo108** [comment]
Hi everyone, any plan for merging this [67](https://github.com/marcglasberg/i18n_extension/pull/67) soon?

**Comment by marcglasberg** [comment]
Sorry, as far as I understand pub.dev doesn't allow different versions for stable and dev. If I accept this change now won't it break the package for 90% of it's users, since they are in stable?

**Comment by nerder** [comment]
@marcglasberg what you should do is to publish a pre-release https://dart.dev/tools/pub/publishing#publishing-prereleases

**Comment by marcglasberg** [comment]
@nerder Got it. I hadn't noticed that pub.dev actually now does allow different versions like this.

**Comment by marcglasberg** [comment]
Published version: `i18n_extension 3.0.0-dev.1`

**Comment by nerder** [comment]
Thank you so much @marcglasberg happy to help
